py-wacz/wacz/main.py

```diff
@@ -118,6 +131,27 @@ def create_wacz(res):
     text_wrap = TextIOWrapper(index_buff, "utf-8", write_through=True)
 
     wacz_indexer = None
+    passed_pages_dict = {}
+
+    if res.pages != None:
+        print("Validate passed Pages")
+        passed_content = open(res.pages, "r").read().split("\n")
+        if passed_content[len(passed_content) - 1] == "":
+            passed_content.pop()
+        validate_passed_pages(passed_content)
```

Comment:
```suggestion
for page_str in passed_content:
  page_json = validate_passed_page(page_str)
  if not page_json:
     # handle invalid
```

I think this can be simplified a bit, just iterate over each line, and then `validaet_passed_page(page_str)` can just try to parse json or return Falsee. The parsed json should already be a dict (otherwise will fail if its not, though could also check)